src/event.rs

```diff
@@ -3,6 +3,25 @@
 //! These are sent to the closure given to [`EventLoop::run(...)`][event_loop_run], where they get
 //! processed and used to modify the program state. For more details, see the root-level documentation.
 //!
+//! Some of these events represent different "parts" of a traditional event-handling loop. You could
+//! approximate the basic ordering loop of [`EventLoop::run(...)`] like this:
+//!
+//! ```rust,no_run
```

Comment:
This is failing the tests, since it doesn't even compile. Change this to `ignore`.